easybuild/easyconfigs/x/XGrafix/XGrafix-2.41-GCCcore-9.3.0.eb

```diff
@@ -0,0 +1,43 @@
+# Contribution from Imperial College London, UK
+# uploaded by J. Sassmannshausen
+
+easyblock = 'ConfigureMake'
+
+name = 'XGrafix'
+version = '2.41'
+
+homepage = 'https://ptsg.egr.msu.edu/'
+description = """A graphics library and controller for physics simulation programs.
+3-d surface plots, scatter plots, 2-d line plots."""
+
+toolchain = {'name': 'GCCcore', 'version': '9.3.0'}
+
+source_urls = ['https://ptsg.egr.msu.edu/pub/codes']
+sources = ['%(namelower)s.tar.gz']
+checksums = ['20daa8f47cb83888e933bb2f451bc8d2b376946b2e7b1ce2252b6e3da0c3b583']
+
+builddependencies = [
+    ('binutils', '2.34'),
+]
+
+dependencies = [
+    ('X11', '20200222'),
+    ('Tcl', '8.6.10'),
+    ('Tk', '8.6.10'),
+    ('ImageMagick', '7.0.10-1'),
+    ('Bison', '3.5.3'),
+]
+
+# We need to specify where things are, unfortunately.
+configopts = '--enable-fulloptimize --with-xpm=$EBROOTX11/lib '
+configopts += '--with-tclconfig=$EBROOTTCL/lib --with-tkconfig=$EBROOTTK/lib'
```

Comment:
should we be including the optional `--with-fortran`?

Reply:
I tried the `--with-fortran` flag for configure and I get:
```
configure: WARNING: unrecognized options: --with-fortran
```
So no, that is not recognised I am afraid.

Reply:
my mistake, I meant `--enable-fortran`

Reply:
Done. The `./configure --help` only showed how to disable it, so I never really looked into that. I also managed to fix the missing `libpng.a` by adding `LIBPNG_PATH=$EBROOTLIBPNG/lib` in the configure line.